content/logs/faq/can-the-datadog-agent-be-used-to-send-only-logs.md

```diff
@@ -35,6 +38,24 @@ The Agent sends metrics and other payloads to Datadog. To make sure that the Age
 3. Configure the Agent to collect logs as explained in the [log documentation page][2].
 4. [Restart the Agent][4]
 
+{{% /tab %}}
+
+{{% tab "Environment Variables" %}}
+
+Set the following environment variables to false:
```

Comment:
I would personally like to backtick `false`

Reply:
not sure what `backtick` means in that case.
Feel free to commit directly on the PR if you want :)